main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	registry := os.Getenv("REGISTRY")
```

Comment:
No strong feeling. We don't tend to prepend the name to env vars for CI tools. Happy to change if a blocker.